docs/ContribOperators.md

```diff
@@ -1186,7 +1186,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 
 <dl>
 <dt><tt>output</tt> : T</dt>
-<dd>3D output tensor with shape (batch_size, sequence_length, v_hidden_size)</dd>
+<dd>3D output tensor with shape (batch_size, num_heads, v_hidden_size)</dd>
```

Comment:
(batch_size, 1, v_hidden_size), since sequence_length == 1 for DMMHA.

Note v_hidden_size is num_heads * v_head_size.

Reply:
Thanks, I misread `v_hidden_size` for `v_head_size`, will revert it.